client/sprotty-ecore/src/features/edit-label-autocomplete.ts

```diff
@@ -0,0 +1,186 @@
+/*******************************************************************************
+ * Copyright (c) 2019 EclipseSource and others.
+ *
+ *   This program and the accompanying materials are made available under the
+ *   terms of the Eclipse Public License v. 2.0 which is available at
+ *   http://www.eclipse.org/legal/epl-2.0.
+ *
+ *   This Source Code may also be made available under the following Secondary
+ *   Licenses when the conditions for such availability set forth in the Eclipse
+ *   Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ *   with the GNU Classpath Exception which is available at
+ *   https://www.gnu.org/software/classpath/license.html.
+ *
+ *   SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ******************************************************************************/
+import { RequestAction, ResponseAction, generateRequestId } from "sprotty/lib";
+import { inject } from "inversify";
+import { matchesKeystroke } from "sprotty/lib/utils/keyboard";
+import { EditLabelUI } from "sprotty/lib";
+import { GLSPActionDispatcher } from "@glsp/sprotty-client/lib";
+import { TYPES } from "@glsp/sprotty-client/lib";
+
+export class AttributeTypesAction  implements RequestAction<ReturnAttributeTypesAction> {
+    static readonly KIND = 'getAttributeTypes';
+    kind = AttributeTypesAction.KIND;
+    constructor(public readonly requestId: string = generateRequestId()) { }
+}
+
+export class ReturnAttributeTypesAction implements ResponseAction {
+    static readonly KIND = 'returnAttributeTypes';
+    kind = ReturnAttributeTypesAction.KIND;
+    types: string[];
+    constructor(public readonly actions: string[], public readonly responseId: string = '') {
+        this.types = actions;
+    }
+}
+
+export class EditLabelUIAutocomplete extends EditLabelUI {
```

Comment:
injectable is an annotation; so it should be `@injectable()` (And not `injectable()` like you had in your previous commit). That's probably why the compiler complained